values.yaml

```diff
@@ -1,3 +1,4 @@
+enable_intermediate_model_removal: False
```

Comment:
what about moving this field under an options field?
It would reflect the Settings dataclass and we could load it easily: `Options(**data['options'])`.

Reply:
Awesome, great idea ! https://github.com/SubstraFoundation/substra-tests/pull/44/commits/274dde387dce5034b0990d20ce51340003a09df4

I also added the `celery_task_max_retry` option while I was there